src/QsCompiler/Tests.Compiler/TestCases/Optimizer/Miscellaneous_output.txt

```diff
@@ -91,43 +91,31 @@ namespace Microsoft.Quantum.Testing {
                 ((Adjoint GlobalCallable CNOT) of ((Unwrap LocalVariable q), (Unwrap LocalVariable b2)[0]));
                 for item in (Unwrap LocalVariable b2) {
                     (GlobalCallable CNOT of ((Unwrap LocalVariable q), LocalVariable item));
-                    (GlobalCallable CNOT of ((Unwrap LocalVariable q), LocalVariable item));
```

Comment:
Why did this change? The test case didn't seem to have changed?

Reply:
The pure circuit optimizer got smarter - it now parses Unwrap expressions as UnknownValues instead of just giving up, so it can cancel the adjacent adjoint gates

Reply:
Which reminds me: Did you ever get around to using the metadata attached to operations to cancel self-adjoint operations?

Reply:
Fixed in latest commit